Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -1571,6 +1574,8 @@ void WebPageProxy::close()
 
     m_provisionalPage = nullptr;
 
+    m_pageForTesting = nullptr;
```

Comment:
Does this mean we need to null check it everywhere?

Reply:
Probably, fixed.